server/settings/workfile_builder.py

```diff
@@ -10,16 +10,18 @@ class CustomBuilderTemplate(BaseSettingsModel):
         default_factory=list,
         title="Task types",
     )
-    template_path: MultiplatformPathModel = SettingsField(
+    path: MultiplatformPathModel = SettingsField(
         default_factory=MultiplatformPathModel
     )
 
 
 class WorkfileBuilderPlugin(BaseSettingsModel):
+    """Legacy use of simple workfile template based on Task type"""
```

Comment:
As mentioned [here](https://github.com/ynput/ayon-aftereffects/issues/29#issuecomment-2740448185) I'm not sure if this is 'legacy' or just simpler. There may be reasons to keep both? No idea.